core/src/tangram.cpp

```diff
@@ -887,6 +887,18 @@ void Map::runAsyncTask(std::function<void()> _task) {
     }
 }
 
+void Map::onMemoryWarning() {
+    auto& tileCache = impl->tileManager.getTileCache();
```

Comment:
Probably data source cache can also be cleared for all loaded datasource?

So:
```c++
for (auto& tileSet : impl->tileManager.getTileSets()) {
    tileSet.source->clearData();
}
```